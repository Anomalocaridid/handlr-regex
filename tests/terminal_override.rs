use insta_cmd::{assert_cmd_snapshot, get_cargo_bin};
use std::process::Command;
use test_case::test_matrix;

#[test_matrix([true, false])]
fn terminal_output_tests_direct(terminal_output: bool) {
    assert_cmd_snapshot!(Command::new(get_cargo_bin("handlr"))
        .arg(format!("--force-terminal-output={terminal_output}"))
        .arg("mime")
        .arg("./assets"))
}
