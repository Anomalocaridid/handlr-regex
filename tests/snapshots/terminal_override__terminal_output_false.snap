---
source: tests/terminal_override.rs
info:
  program: handlr
  args:
    - "--force-terminal-output=false"
    - mime
    - "./assets"
---
success: true
exit_code: 0
----- stdout -----
path    	mime                  
./assets	application/x-zerosize

----- stderr -----
